pallets/rc-migrator/src/accounts.rs

```diff
@@ -536,6 +534,11 @@ impl<T: Config> AccountsMigrator<T> {
 		Ok(Some(withdrawn_account))
 	}
 
+	/// Actions to be done after the accounts migration is finished.
+	pub fn finish_balances_migration() {
+		pallet_balances::InactiveIssuance::<T>::put(0);
```

Comment:
Afaik, `InactiveIssuance` affects support for governance conviction voting and the deactivate/reactivate is also related to the crowdloans and treasury.

Can we add here some comment/reasoning why we need this set to `0` here? And why we need to do it after account_balances migrations and not after governance/treasury or crowdloans migration?

And another question, why is this not migrated similar as total issuance [here](https://github.com/polkadot-fellows/runtimes/pull/607)?

Reply:
this is RC side, on RC there wont be any `InactiveIssuance`, its moved to AH. On AH instead of setting the RC value (which is actually wrong now) we recalculate it.